osh/hub/scan/notify.py

```diff
@@ -291,7 +295,7 @@ def send_scan_notification(request, scan_id):
         "X-Scan-Package": scan.package.name,
         "X-Scan-Build": scan.nvr,
     }
-    return send_mail(message, subject, [recipient], headers)
+    return send_mail(message, subject, recipients, headers)
```

Comment:
recipients is a `list` or an `str`, why need to change it to a `set`

Reply:
I copied the code from a [similar place](https://github.com/openscanhub/openscanhub/blob/6e1fb9f774fde5bbf094d0ac9ac6b3c71b1fb1e7/osh/hub/scan/notify.py#L144). I would try to fix it now.